OpenSim/Common/XMLDocument.cpp

```diff
@@ -54,7 +54,8 @@ using namespace std;
 // 30502 for Changing serialization of Geometry
 // 30503 for Changing serialization of Ground
 // 30505 for Changing serialization of Joint to create offset frames
-const int XMLDocument::LatestVersion = 30505;   
+// 30506 for Changing serialization of PrescribedForce to use Connectors
+const int XMLDocument::LatestVersion = 30506;   
```

Comment:
Here's the issue with not upgrading version number. If you save the file with the old version number, then the contents will be inconsistent with the version number in the header and there will be no way to tell that the file has been partially updated. the updatefromXML code will likely get triggered again which may try to recreate connections or duplicate xml structure.

Usually the code in updateFromXML is guarded by a check that the version is less than latest to avoid the conversion altogether on files conforming to the latest format. So if we don't bump we'll be putting  off many bugs related to conversion issues downstream until we bump the version number but I don't have a concrete example to back this up.